docs/README.md

```diff
@@ -142,7 +142,7 @@ Frameworks:
 - Flexium: [FlexGUI](https://github.com/DXsmiley/FlexGUI)
 - GML/GameMakerStudio2: [ImGuiGML](https://marketplace.yoyogames.com/assets/6221/imguigml)
 - Irrlicht: [IrrIMGUI](https://github.com/ZahlGraf/IrrIMGUI)
-- Ogre: [ogreimgui](https://bitbucket.org/LMCrashy/ogreimgui/src)
+- Ogre: [ogre-imgui](https://github.com/OGRECave/ogre-imgui)
 - OpenFrameworks: [ofxImGui](https://github.com/jvcleave/ofxImGui)
```

Comment:
You removed link to the 
https://bitbucket.org/LMCrashy/ogreimgui/src
Repo.
Is that new link an evolution/rename of the old one?

Reply:
yes

Reply:
I changed the link to point to the new location of the referenced repository